app/models/user.rb

```diff
@@ -1,13 +1,63 @@
 class User < Base
+  include AASM
+
   has_many :organisation_users
   has_many :organisations, through: :organisation_users
 
-  custom_endpoint :accept_transition_screen, on: :member, request_method: :patch
-  custom_endpoint :accept_rollover_screen, on: :member, request_method: :patch
   custom_endpoint :accept_terms, on: :member, request_method: :patch
 
-  def self.member(id)
-    new(id: id)
+  module State
+    NEW = "new".freeze
+    TRANSITIONED = "transitioned".freeze
+    ROLLED_OVER = "rolled_over".freeze
+    NOTIFICATIONS_CONFIGURED = "notifications_configured".freeze
+  end
```

Comment:
The only place these are used is to check the state the user is in, which is better handled by  [`aasm`'s built-in predicates](https://github.com/aasm/aasm#usage) anyway, and in specs. Do we really need it?

Reply:
👍

Reply:
Yep - they are only being used for the purposes of testing. I'll remove